hc/api/tests/test_badge.py

```diff
@@ -13,12 +13,17 @@ def setUp(self):
 
     def test_it_rejects_bad_signature(self):
         r = self.client.get("/badge/%s/12345678/foo.svg" % self.alice.username)
-        ### Assert the expected response status code
+        ### Assert the expected response status code -- DONE
+        assert r.status_code == 400
```

Comment:
You should use `self.assertEqual`. e.g `self.assertEqual(400, r.status_code)`. Note that it's common and a best practice to start with the expected value followed by the returned value. https://stackoverflow.com/questions/9331259/unit-testing-why-is-the-expected-argument-always-first-in-equality-tests